x-pack/plugins/canvas/canvas_plugin_src/renderers/advanced_filter/component/advanced_filter.tsx

```diff
@@ -4,12 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import PropTypes from 'prop-types';
+import React, { SFC } from 'react';
```

Comment:
Minor: I think the name `SFC` got deprecated - in the code, I see a mix of `React.StatelessComponent` and `React.SFC`, slightly preferring the former. I tried but couldn't directly hop to the new name `FunctionComponent`, not sure why as we upgraded to `v16.6`, haven't looked further as you likely know.

Reply:
It has been, but `@types/react` has not been updated yet.  We'll have to wait until it is until we change this... should be relatively soon.